starter-code/routes/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const router  = express.Router();
+
+/* GET home page */
+router.get('/', (req, res, next) => {
+  res.render('index');
+});
+
+/* C(R)UD: Retrieve -> Listing Our Celebrities */
+router.get('/celebrities',(req, res, next)=> {
+  Celebrity.find({}).then(celebrities => {
+    res.render('celebrities',{celebrities});
```

Comment:
`res.render('celebrities/index',{celebrities});`
en la carpeta view el index de la celebrities es in celebrities/index